### ISSUE TITLE
Add warning level to configuration options

### ISSUE THREAD

**Issue by npetkov** [mmozuras/poper#9]
It's great that poper now supports external config files. I have a hopefully simple feature request - could you please make the warning level of the messages configurable as well? This is desirable in cases where ill-formatted commit messages have already slipped upstream and would break CI builds configured with pronto's exit-code option.

**Comment by ivanovaleksey** [comment]
@npetkov could you provide some example?
You can specify how much commits you want to run a check on.

**Comment by npetkov** [comment]
Yes, I'll try to clarify what I mean. Consider the [check method](https://github.com/mmozuras/poper/blob/ee9026aa7741ce0a0cd98c2180480ece87b788b6/lib/poper/runner.rb#L17). It returns an array of objects:

```
OpenStruct.new(commit: commit.oid, message: result) if result
```

So essentially, every "error" returned from `poper` will have a `commit` and `message` field. I think it would be useful if another field is added, called `:level` (in line with `pronto` naming), which is globally configurable via the configuration file:

```
OpenStruct.new(commit: commit.oid, message: result, level: [this should be configurable]) if result
```

This configurable "severity" is useful for certain cases where "non-compliant" commit messages (like merge commits) are unavoidable.

Of course, this would require a change in the `poper-runner` [main method](https://github.com/prontolabs/pronto-poper/blob/5630ce58e7f745c96afd68e7140aeeda7fdeb4f9/lib/pronto/poper.rb#L16) as well, but this is a matter of another issue:

```
def message_for(error)
  Message.new(nil, nil, error.level, error.message.capitalize, error.commit, self.class)
end
```

I hope this makes more sense now.
